src/credentials/CHIPCert.h

```diff
@@ -967,6 +738,15 @@ CHIP_ERROR ExtractNodeIdFabricIdFromOpCert(const ByteSpan & opcert, NodeId * nod
  */
 CHIP_ERROR ExtractPublicKeyFromChipCert(const ByteSpan & chipCert, P256PublicKeySpan & publicKey);
 
+/**
+ * Extract Not Before Time from a chip certificate in ByteSpan TLV-encoded form.
+ * This does not perform any sort of validation on the certificate structure
+ * other than parsing it.
```

Comment:
Should probably document that the time is a CHIP Epoch time, and that 0 represents "not defined"?